### ISSUE TITLE
pip install not working

### ISSUE THREAD

**Issue by naturalethic** [Dwolla/dwolla-v2-python#2]
```
Collecting dwollav2
  Using cached dwollav2-1.1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/cg/v9dhxy1x42j_q_zsfgb1blsh0000gn/T/pip-build-0mANaF/dwollav2/setup.py", line 23, in <module>
        long_description=open('README.md').read(),
    IOError: [Errno 2] No such file or directory: 'README.md'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/cg/v9dhxy1x42j_q_zsfgb1blsh0000gn/T/pip-build-0mANaF/dwollav2/
```

**Comment by sausman** [comment]
Hey @naturalethic,

Thanks for the heads up. I just merged a potential fix (1.1.1). Let me know if that doesn't fix it.

**Comment by naturalethic** [comment]
Looks good. Thanks!
